flows/definition/flow.go

```diff
@@ -28,7 +28,7 @@ const (
 )
 
 // CurrentSpecVersion is the flow spec version supported by this library
-var CurrentSpecVersion = semver.MustParse("13.6.1")
+var CurrentSpecVersion = semver.MustParse("13.7.0")
```

Comment:
no migration required but we are technically introducing a new action so flows with that aren't backward compatible